magento/pwa-studio
# Spectrum redesign

This pull request (PR) introduces Spectrum design system and CSS components.



## Description

Introduces the new theme version and changes some layout files to accomodate to new design system.



## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the docs site.
2. See the new design.

## Screenshots / Screen Captures (if appropriate)

<img width="1424" alt="Screen Shot 2020-05-06 at 1 48 02 PM" src="https://user-images.githubusercontent.com/752610/81216241-3b42e300-8fa0-11ea-8420-5f974b49bf11.png">

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
